identity/did.go

```diff
@@ -108,6 +108,26 @@ type DID common.Address
 // DIDLength contains the length of a DID
 const DIDLength = common.AddressLength
 
+// MarshalJSON marshals DID to json bytes.
+func (d DID) MarshalJSON() ([]byte, error) {
+	str := "\"" + d.String() + "\""
```

Comment:
why we need the quotes?

Reply:
It is the JSON standard way of serializing as string vs other types.